### PR TITLE
Add linked templates when querying show_template

### PR DESCRIPTION
**Pull request by linceaerian**
In addition to hosts, It's useful to have the list of child template linked too.

### CODE REVIEW

**Comment by paalbra** [comment]
I don't use linked templates much myself as I find them a bit hard to manage, but this seems like a nice to have feature!

But the web frontend (4.0) only displays parent templates in the "Linked templates" view. I'm curious why did you choose to display the child templates? Maybe change this to parent templates or select both child and parent templates in two separate columns?

Also I would use list comprehension rather than your append loop. I believe that this would add too readability. https://docs.python.org/3.7/tutorial/datastructures.html#list-comprehensions

**Comment by linceaerian** [comment]
@paalbra In fact it's because I wanted to check the usage of templates (How much hosts and templates depends of the current checked template).
And yes, the 4.0 removed the linked hosts & child template from the template page, and move the view to the templates page (host are in light green and child templates in light grey).

About the optimisation, I have to say I'm still not fluent enough in python to do it like you said *^^* so I did it the way I could :)

File: zabbix_cli/cli.py [hunk 1]
```diff
@@ -4869,18 +4871,25 @@ def do_show_template(self, args):
...
             result_columns_key = result_columns_key + 1
 
         #
         # Generate output
         #
         self.generate_output(result_columns,
-                             ['TemplateID', 'Name', 'Hosts'],
-                             ['Name', 'Hosts'],
+                             ['TemplateID', 'Name', 'Hosts','Templates'],
+                             ['Name', 'Hosts','Templates'],
```
Comment (paalbra) [inline zabbix_cli/cli.py]: Whitespace missing before 'Templates' in these two lists

**Comment by paalbra** [comment]
I noticed that other parts of the code also uses a similar "append loop", so I guess it's fine for now.

Could you fix the whitespace error I've commented on? It produces linting errors

**Comment by linceaerian** [comment]
Hello @paalbra 

I corrected the missing whitespace  :)

**Comment by paalbra** [comment]
Cool. Merged

**Pull request merged**